service/handlers.go

```diff
@@ -1267,8 +1267,11 @@ func generateEnvVariablesString(environmentVariables map[string]string) string {
 
 	r := "env"
 	for k, v := range environmentVariables {
-		tmp := r + fmt.Sprintf(" %s=%s", k, v)
-		r = tmp
+		if strings.HasPrefix(v, "'") {
```

Comment:
we should take a different approach here.  

1. remove any single and double quotes
2. then surround the value in single quotes

things will get weird their are errant single quotes or anything mid string.

Reply:
quite not sure whether we need to remove any single or double quotes if single or double quotes are part of environmental variables.

Reply:
updated